source/includes/fact-retryable-writes-election.rst

```diff
@@ -0,0 +1,10 @@
+Your application connection logic should include tolerance for 
+elections, such as queueing write operations until it detects a healthy 
+primary.
```

Comment:
Can you explain what you mean by queuing write operations until it detects a healthy primary? and how that interacts with the retryable writes?